### ISSUE TITLE
Unable to create a new entity for jhipster 6.8.0 bug

### ISSUE THREAD

**Issue by khashvin** [jhipster/generator-jhipster-nodejs#147]
**Describe the bug**
Unable to create a second entity after starting a new project
This is the error that I got after creating the second entity
```
events.js:174
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read property 'toLowerCase' of undefined
    at Function.plural (C:\Users\HASHVIN\Documents\lov-admin\node_modules\pluralize\pluralize.js:133:24)
    at pluralize (C:\Users\HASHVIN\Documents\lov-admin\node_modules\pluralize\pluralize.js:174:46)
    at context.relationships.forEach.relationship (C:\Users\HASHVIN\Documents\lov-admin\node_modules\generator-jhipster\generators\entity\index.js:933:74)
    at Array.forEach (<anonymous>)
    at module.exports.loadInMemoryData (C:\Users\HASHVIN\Documents\lov-admin\node_modules\generator-jhipster\generators\entity\index.js:877:39)
    at Object.<anonymous> (C:\Users\HASHVIN\Documents\lov-admin\node_modules\yeoman-generator\lib\index.js:647:25)
    at C:\Users\HASHVIN\Documents\lov-admin\node_modules\run-async\index.js:49:25
    at new Promise (<anonymous>)
    at C:\Users\HASHVIN\Documents\lov-admin\node_modules\run-async\index.js:26:19
    at self.env.runLoop.add.completed (C:\Users\HASHVIN\Documents\lov-admin\node_modules\yeoman-generator\lib\index.js:648:11)
Emitted 'error' event at:
    at Immediate.setImmediate (C:\Users\HASHVIN\Documents\lov-admin\node_modules\yeoman-generator\lib\index.js:668:20)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```

**To Reproduce**

1. Create new project
2. Create 1st entity with entity generator
3. Create 2nd entity with many to one relationship to 1st entity

**Desktop (please complete the following information):**

-   OS: Windows 10
-   NodeJS v10.20.1

**Comment by amanganiello90** [comment]
Hi @khashvin , thanks for the feedback!! But could you please add the steps command that you have executed with the entity name? maybe you have choosen a name not allowed.

**Comment by amanganiello90** [comment]
I investigate on this, it is a jhipster bug fixed in 6.9.0 version : https://github.com/jhipster/generator-jhipster/issues/11595

So we need to upgrade the version from 6.8.0.
Thanks.
